config/core/test-configs.yaml

```diff
@@ -228,6 +228,12 @@ test_plans:
     filters:
       - passlist: {defconfig: ['kselftest']}
 
+  kselftest_a_qemu: &kselftest_qemu
```

Comment:
What's the `a` for? Alphabetical order?

Reply:
Yes, like the commit log says it was just a bodge so it could stay at the top of the list of kselftests.

Reply:
Ah yes sorry, the PR description only has half of the actual commit message (guess that was an update).  Might be worth a comment in YAML though about that.

Reply:
Added a comment.